zerver/tests/test_messages.py

```diff
@@ -467,12 +468,19 @@ def send_message():
             check_send_message(sender, sending_client, message_type_name, [stream],
                                subject, content, forwarder_user_profile=sender, realm=realm)
 
-        flush_per_request_caches()
         send_message() # prime the caches
```

Comment:
We should remove the line above--note that it says "prime the caches."  I think the original intent of the code was to prime the caches so we had a somewhat "real world" execution of the query, and I think back in the day we set a max number of queries.  Then, for a while, I think things were deterministic in a non-parallel world.  I could be wrong about the latter.

Anyway, I don't think the original intent is important to preserve here--we can delete the line.  But it's food for thought.